### PR TITLE
Production env

### PR DESCRIPTION
**Pull request by sujeebant**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -7,8 +7,14 @@ const pump = require('pump')
...
 const promiseHyperdb = require('./promiseHyperdb')
 
-const HUB_URL = 'localhost:8080'
-const MASQ_APP_BASE_URL = 'http://localhost:3000'
+const MASQ_APP_BASE_URL = (function () {
+  switch (process.env.NODE_ENV) {
+    case ('development'):
+      return 'http://localhost:3000'
+    default:
+      return 'https://qwantresearch.github.io/masq-app/'
```
Comment (G-Ray) [inline src/index.js]: We should not hardcode these values. Can you add a a config.js file where we define dev/pro urls ?
Maybe we will need to set others values too, so maybe create a folder with 2 configs: dev.config.js, and prod.config.js, and the correct one will be loaded depending on the `NODE_ENV`

Comment (sujeebant) [reply src/index.js]: done

File: src/index.js [hunk 2]
```diff
@@ -4,12 +4,10 @@ const rai = require('random-access-idb')
 const hyperdb = require('hyperdb')
 const uuidv4 = require('uuid/v4')
 const pump = require('pump')
+const fs = require('fs')
```
Comment (G-Ray) [inline src/index.js]: Nope, you should not require `fs`, do as you have done with masq-demos, require the correct config dynamically

File: config/dev.config.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "MASQ_APP_BASE_URL": "http://localhost:3000"
+}
```
Comment (G-Ray) [inline config/dev.config.json]: You should  configure your editor to add newline

File: config/dev.config.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (G-Ray) [inline config/dev.config.json]: don't you prefer `config.dev.js`, that sound like `webpack.dev.js`

File: src/index.js (hunk 2 above)
Comment (sujeebant) [reply src/index.js]: done

File: config/dev.config.json (hunk 4 above)
Comment (sujeebant) [reply config/dev.config.json]: changed

File: config/dev.config.json (hunk 3 above)
Comment (sujeebant) [reply config/dev.config.json]: done :+1:

**Pull request merged**